src/lib/settings/initialSettings.tsx

```diff
@@ -0,0 +1,383 @@
+          options: (cmdContext, settingsContext) =>
+            Object.values(cameraSystems).map((v) => ({
+              name: v,
+              value: v,
+              isCurrent:
+                v ===
+                settingsContext.modeling.mouseControls[
+                  cmdContext.argumentsToSubmit.level as SettingsLevel
+                ],
+            })),
+        },
+        Component: ({ value, onChange }) => (
+          <>
+            <select
+              id="camera-controls"
+              className="block w-full px-3 py-1 bg-transparent border border-chalkboard-30"
+              value={value}
+              onChange={onChange}
+            >
+              {cameraSystems.map((program) => (
+                <option key={program} value={program}>
+                  {program}
+                </option>
+              ))}
+            </select>
+            <ul className="mx-0 my-2 flex flex-col gap-2 text-sm">
+              <li className="grid grid-cols-4 gap-1">
+                <strong>Pan</strong>
+                <p className="col-span-3 leading-tight">
+                  {cameraMouseDragGuards[value].pan.description}
+                </p>
+              </li>
+              <li className="grid grid-cols-4 gap-1">
+                <strong>Zoom</strong>
+                <p className="col-span-3 leading-tight">
+                  {cameraMouseDragGuards[value].zoom.description}
+                </p>
+              </li>
+              <li className="grid grid-cols-4 gap-1">
+                <strong>Rotate</strong>
+                <p className="col-span-3 leading-tight">
+                  {cameraMouseDragGuards[value].rotate.description}
+                </p>
+              </li>
+            </ul>
+          </>
+        ),
+      }),
+      /**
+       * Whether to show the debug panel, which lets you see
+       * various states of the app to aid in development
+       */
+      showDebugPanel: new Setting<boolean>({
+        defaultValue: false,
+        description: 'Whether to show the debug panel, a development tool',
+        validate: (v) => typeof v === 'boolean',
+        commandConfig: {
+          inputType: 'boolean',
+        },
+      }),
+      /**
+       * TODO: This setting is not yet implemented.
+       * Whether to turn off animations and other motion effects
+       */
+      // reduceMotion: new Setting<boolean>({
+      //   defaultValue: false,
+      //   description: 'Whether to turn off animations and other motion effects',
+      //   validate: (v) => typeof v === 'boolean',
+      //   commandConfig: {
+      //     inputType: 'boolean',
+      //   },
+      //   hideOnLevel: 'project',
+      // }),
+      /**
+       * TODO: This setting is not yet implemented.
+       * Whether to move to view the sketch plane orthogonally
+       * when creating entering or creating a sketch.
+       */
+      // moveOrthoginalToSketch: new Setting<boolean>({
```

Comment:
Actually similar to my last point, what got discussed as part of `https://github.com/KittyCAD/engine/issues/1874`
Should probably have three settings not boolean, they would be along the lines of:
- don't move camera
- move camera and align camera-up to the nearest most upwardly aligned sketch axis
- move camera and always align camera-up to the positive sketch-y-axis

But needs word smithing for sure

Reply:
For sure, and we can provide descriptions for each of the options by using a custom component like I did for the `cameraControls` setting, so the name of the values doesn't have to do so much heavy lifting.